fabric-common/lib/BlockDecoder.js

```diff
@@ -615,13 +615,15 @@ rule
 		if (!(processedTransactionBuf instanceof Buffer)) {
 			throw new Error('Processed transaction data is not a byte buffer');
 		}
-		const processed_transaction = {};
```

Comment:
Could we leave this holder of the return object, that way the reader will know what is being built to be returned within the method.

Reply:
@harrisob  I assumed reader could get enough information from heading jsDoc,
` @returns {ProcessedTransaction} A fully decoded ProcessedTransaction object`